### PR TITLE
Create announcement and event post for makerfestival

### PR DESCRIPTION
**Pull request by dcwalk**
Should get something up ASAP and actually start advertising 📦 
@garrying or @benhylau can you do a quick rev?

### CODE REVIEW

File: _announcements/2016-07-11-makerfestival.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Makerfestival"
+dateUntil: 2016-07-11
+---
+
+Come speak with us as this year's [Makerfestival](http://makerfestival.ca)! We'll be hosting a **FREE** satellite event **"Introduction to Networks: From the Internet to tomesh.net"** on June, 25 at 1pm (email [hello@tomesh.net](mailto:hello@tomesh.net) to register or [find out more]()) and have a table at the [festival itself](http://makerfestival.ca) on July 9-10!
```
Comment (benhylau) [inline _announcements/2016-07-11-makerfestival.md]: `[find out more]()` missing link?

**Comment by benhylau** [comment]
Was just thinking about this. Thanks @dcwalk !

File: _announcements/2016-07-11-makerfestival.md (hunk 1 above)
Comment (dcwalk) [inline _announcements/2016-07-11-makerfestival.md]: Just added -- made a change to how permalinks are generated (in config), added a slug (on post) and then added the link (in announcement)... hahah did I over-complicate this much?!

**Comment by benhylau** [comment]
:+1:

**Comment by dcwalk** [comment]
@benhylau thanks for the quick look! I think it is ready to merge. Just had to do a wee bit of last wrangling (sorry, late and tired over here!)

**Pull request merged**